pkg/test/cel/httproute_experimental_test.go

```diff
@@ -432,3 +432,48 @@ func TestHTTPRouteRuleExperimental(t *testing.T) {
 		})
 	}
 }
+
+func TestHTTPRequestMirrorFilterExperimental(t *testing.T) {
```

Comment:
Would be great to create a corresponding `grpcroute_experimental_test` file and have the same tests run for GRPCRoute.

Reply:
Done! Let me know if I missed anything.

Reply:
Oh I forgot to make the new test use grpcroute 🤦 one moment